examples/contiki/coap-observer.c

```diff
@@ -140,7 +140,9 @@ PROCESS_THREAD(coap_server_process, ev, data)
   if (uri.port != COAP_DEFAULT_PORT) {
     unsigned char portbuf[2];
     coap_add_option(request, COAP_OPTION_URI_PORT,
-		    coap_encode_var_bytes(portbuf, uri.port), portbuf);
+		    coap_encode_var_safe(portbuf, sizeof(portbuf),
+                                         uri.port),
```

Comment:
It seems that your editor adds a leading white space for wrapped argument lists (multiple occurrences follow).

Reply:
Hmm - that is because TAB has been used instead of SPACE * 8 - done previous to my updates - e.g. ^I is the TAB, $ is end of line  -

```
  if (uri.port != COAP_DEFAULT_PORT) {$
    unsigned char portbuf[2];$
    coap_add_option(request, COAP_OPTION_URI_PORT,$
^I^I    coap_encode_var_safe(portbuf, sizeof(portbuf),$
                                         uri.port),$
                    portbuf);$
  }$
```

Looking at the code as patched, all is aligned, so I missed this.  There are lots of places where TAB has been used it seems.